### ISSUE TITLE
Disable to capitalization and auto punctuation.

### ISSUE THREAD

**Issue by Doubs115** [futo-org/voice-input#35]
Currently, it just likes to place periods and capitals wherever it wants. Realistically, sometimes people want to replace certain words that they don't know how to type. So therefore, we should have the option to either disable periods, capitals, and comma. Either individually or all at the same time.

I often find myself replacing capitals and periods often but not so for commas. I bet with better training of these models over time they will be able to figure it out but right now it's not good for this.

Another addition, is maybe to have it scan for the words around it to get better context and to place punctuation accordingly.

**Comment by derjohn** [comment]
+1 from /me. I tried to dictate the punctuation but the modes writes it as word, not as punctuation character (Like "Exclamation Mark" instead of "!")

**Comment by David-Prock** [comment]
Hello, ellipsis.

**Comment by WofWca** [comment]
I don't care as much about punctuation, but capitalization important for me as to someone who does edit pre-existing text often.

My suggestion would be not to completely disable capitilization (altgough that could also be a toggle),
but try to detect whether the text from voice input is to be inserted in the middle of a sentence.

**Comment by BeegMan27** [comment]
Yes! This behavior is absolutely ridiculous, even when I already have the option to allow dictation of punctuation, it still peppers commas around where the grammar does not call for them. Moreover, if I dictate a three-word sentence fragment, I do not need sentence-final punctuation to be automatically inserted. If the system isn't smart enough to place punctuation where it is appropriate in the sentence, then is should not be putting anything in at all.
